ABloom/ABloom/Resources/Extensions/Ex+Text.swift

```diff
@@ -16,8 +16,10 @@ extension Text {
    -
    .fontWithTracking(fontStyle: .largeTitleBold, value: 1)
    */
-  func fontWithTracking(fontStyle: Font, value: CGFloat = 1) -> Text {
-    self.font(fontStyle)
+  func fontWithTracking(fontStyle: Font, value: CGFloat = 1) -> some View {
```

Comment:
```suggestion
  func fontWithTracking(fontStyle: Font, tracking value: CGFloat = 1) -> some View {
```

이런식으로 써도 좋을 것 같아요! 메서드를 사용하는 입장에서 value가 무엇인지 명확하지 않으니, 함수 내부에서는 value로 써도 괜찮지만, 외부에 보이는 파라미터 값은 명확하게 보이는 것이 좋을 듯 합니다.

Reply:
좋습니다!! 여기서 수정하면 다른 곳에서 에러가 발생할것 같아, 다음 PR에서 정리해서 올리겠습니다!